org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/server/Server.java

```diff
@@ -307,7 +309,14 @@ public void stop() {
 
   @Override
   public synchronized void updateStorage(IProgressMonitor monitor) {
-    updateStatus = client.update(getConfig(), new WrappedProgressMonitor(monitor, "Update configuration from server '" + getId() + "'")).status();
+    UpdateResult updateResult = client.update(getConfig(), new WrappedProgressMonitor(monitor, "Update configuration from server '" + getId() + "'"));
+    for (SonarAnalyzer analyzer : updateResult.analyzers()) {
+      if (!analyzer.sonarlintCompatible() && analyzer.minimumVersion() != null) {
```

Comment:
This doesn't seem correct. Check [SLI](https://github.com/SonarSource/sonarlint-intellij/blob/master/src/main/java/org/sonarlint/intellij/tasks/ServerUpdateTask.java#L95)